File: pgm/ABC_Java/ABC_212_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final int A = Integer.parseInt (arr[0]);
        final int B = Integer.parseInt (arr[1]);
        final int zero = 0;
        final String Gold = "Gold";
        final String Silver = "Silver";
        final String Alloy = "Alloy";
        if (A > zero && B > zero) {
            System.out.println (Alloy);
        } else if (A == zero) {
            System.out.println (Silver);
        } else {
            System.out.println (Gold);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_212_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer
// import java.util.TreeMap

// import kotlin.math.*

const val Gold: String = "Gold"
const val Silver: String = "Silver"
const val Alloy: String = "Alloy"
const val space: String = " "
const val zero: Int = 0

fun main() {
    val (A, B) = readln().split(space).map { it.toInt() }
    if (A > zero && B > zero) {
        println(Alloy)
    } else if (A == zero) {
        println(Silver)
    } else {
        println(Gold)
    }
    kotlin.system.exitProcess(0)
}